upi/vsphere/vm/main.tf

```diff
@@ -29,8 +30,9 @@ resource "vsphere_virtual_machine" "vm" {
   }
 
   extra_config = {
-    "guestinfo.ignition.config.data"          = base64encode(data.ignition_config.ign[each.key].rendered)
-    "guestinfo.ignition.config.data.encoding" = "base64"
+    "guestinfo.afterburn.initrd.network-kargs" = "ip=${each.value}::${cidrhost(machine_cidr, 1)}:255.255.255.0::ens192:off"
```

Comment:
@LorbusChris the mask isn't correct. Should be: `255.255.254.0`

Reply:
@jcpowermac thanks for catching that! updated